contributors/devel/cri-testing-policy.md

```diff
@@ -0,0 +1,118 @@
+# Container Runtime Interface: Testing Policy
+
+**Owner: SIG-Node**
+
+This document describes testing policy and process for runtimes implementing the
+[Container Runtime Interface (CRI)](/contributors/devel/container-runtime-interface.md)
+to publish test results in a federated dashboard. The objective is to provide
+the Kubernetes community an easy way to track the conformance, stability, and
+supported features of a CRI runtime.
+
+This document focuses on Kubernetes node/cluster end-to-end (E2E) testing
+because many features require integration of runtime, OS, or even the cloud
+provider. A higher-level integration tests provider better signals on vertical
+stack compatibility to the Kubernetes community. On the other hand, runtime
+developers are strongly encouraged to run low-level
+[CRI validation test suite](https://github.com/kubernetes-incubator/cri-tools/blob/master/docs/validation.md)
+for validation as part of their development process.
+
+## Required and optional tests
+
+Runtime maintainers are **required** to submit the tests listed below.
+ 1. Node conformance test suite
+ 2. Node feature test suite
```

Comment:
@yujuhong One thing I want to amend is, for hypervisor based runtimes, it actually need run in mixed runtime mode to pass all node conformance tests. We may need to mention this somewhere?

Mixed runtime mode can be like: Kata + Docker, Kata + containerd etc. They will expose one endpoint to kubelet or use a cri-proxy.

Reply:
IMO, the runtime maintainers can decide whether they are going to use proxy or not, but that seems immaterial to the policy.
Pure hypervisor-based runtime is not going to pass all the tests, but the test results will still provide useful signals to the community. I think as a SIG, we can discuss case-by-case, when it comes up, to tolerate more failures for these non-traditional runtimes.